These 2 flags cause the following errors when linking:

  /bin/ld: error: /tmp/ccTsLi3n.ltrans0.ltrans.o: requires dynamic R_X86_64_PC32 reloc against '_ZN16AutoTypeExecutor14execClearFieldEP18AutoTypeClearField' which may overflow at runtime; recompile with -fPIC

Since AOSC has harden flags opened in GCC specs these 2 flags can be removed.

Junde Yhi
--- a/CMakeLists.txt	2017-09-05 23:21:04.378138696 +0800
+++ b/CMakeLists.txt	2017-09-05 23:22:43.547674426 +0800
@@ -140,7 +140,6 @@
   if (CMAKE_COMPILER_IS_CLANGXX)
       add_gcc_compiler_flags("-Qunused-arguments")
   endif()
-  add_gcc_compiler_flags("-pie -fPIE")
   set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -Wl,--no-add-needed -Wl,--as-needed -Wl,--no-undefined")
   set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -Wl,-z,relro,-z,now")
   set(CMAKE_MODULE_LINKER_FLAGS "${CMAKE_MODULE_LINKER_FLAGS} -Wl,--no-add-needed -Wl,--as-needed")
